introduction.md

```diff
@@ -2,3 +2,8 @@ Hi,
 I'm eabrahamo and I'm following the Mastering GitHub course on codeschool. 
 
 My favorite path is the Electives path
+
+Deadly skills:
+* Java
+* SDK
+* HTML
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?